apps/web/src/services/asset-inventory/collector/shared/collector-data-modal/modules/CollectorDataDuplicationInner.vue

```diff
@@ -70,19 +70,23 @@ import { PDefinitionTable, PI, PStatus } from '@spaceone/design-system';
 
 import { coral, green } from '@/styles/colors';
 
+import type { CollectorData } from '@/services/asset-inventory/collector/shared/collector-data-modal/type';
 import { ACCOUNT_TYPE } from '@/services/asset-inventory/collector/shared/collector-data-modal/type';
 
 const SUCCEEDED_COLOR = green[400];
 const FAILED_COLOR = coral[400];
 
 interface Props {
     accountType?: string;
+    item?: CollectorData;
```

Comment:
question: Why is this needed? It doesn't seem to be used anywhere.🧐

Reply:
Oh, it seems like it will be used later when real data comes in.

Reply:
It will probably be necessary when connecting real data!